code/modules/power/singularity/particle_accelerator/particle_accelerator.dm

```diff
@@ -122,18 +122,18 @@ So, hopefully this is helpful if any more icons are to be added/changed/wonderin
 	return 1
 
 /obj/structure/particle_accelerator/examine(mob/user)
+	. = ..()
 	switch(construction_state)
-		if(0)
-			desc = text("A [name], looks like it's not attached to the flooring")
-		if(1)
-			desc = text("A [name], it is missing some cables")
-		if(2)
-			desc = text("A [name], the panel is open")
-		if(3)
-			desc = text("The [name] is assembled")
+		if(ACCELERATOR_UNWRENCHED)
+			. += "<span class='notice'>The [name]'s anchoring bolts are <i>loose</i>. It can be rotated with Alt-Click.</span>"
```

Comment:
Loose doesn't really explain what needs to be done here, maybe highlight the bolts instead? Probably also best to have Alt-Click on the last line also.